crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -375,71 +376,76 @@ pub fn check_visibility(
         let view_mask = maybe_view_mask.copied().unwrap_or_default();
 
         visible_entities.entities.clear();
```

Comment:
Wondering if we could just combine the two queries together instead of splitting them like this.

Reply:
Tested this over here: https://github.com/bevyengine/bevy/pull/10196